azurerm/internal/services/netapp/tests/resource_arm_netapp_volume_test.go

```diff
@@ -366,6 +372,10 @@ resource "azurerm_netapp_volume" "test" {
     unix_read_only  = true
     unix_read_write = false
   }
+
+  tags = {
+    "foo" = "bar"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```